go.mod

```diff
@@ -1,74 +1,78 @@
 module github.com/ipfs/go-unixfsnode
 
-go 1.21
+go 1.22
```

Comment:
sorry, I merged the web3-bot auto PR, I should have closed that one in favour of this, but you'll have a conflict to resolve now